lib/chat/new_matches.dart

```diff
@@ -1,82 +1,75 @@
-import 'package:google_fonts/google_fonts.dart';
+import 'package:flutter_redux/flutter_redux.dart';
 import 'package:flutter/material.dart';
+import 'package:redux/redux.dart';
 
+import '../common/format_text.dart';
 import '../constants/chat_constants.dart';
+import '../models/models.dart';
```

Comment:
Unused import statement.

Reply:
AppState is being used from here?

Reply:
For some reason, my editor said it wasn't being used. 🤷🏾‍♀️

Reply:
It's not being used in `message.dart`, not this one hehe

<img width="298" alt="Screen Shot 2020-07-28 at 8 53 47 PM" src="https://user-images.githubusercontent.com/25041263/88751250-70c91980-d114-11ea-804d-aeaeeff03668.png">